packages/edit-site/src/components/sidebar-edit-mode/index.js

```diff
@@ -22,27 +25,79 @@ import TemplatePanel from './template-panel';
 import PluginTemplateSettingPanel from '../plugin-template-setting-panel';
 import { SIDEBAR_BLOCK, SIDEBAR_TEMPLATE } from './constants';
 import { store as editSiteStore } from '../../store';
+import { unlock } from '../../lock-unlock';
+
+const { Tabs } = unlock( componentsPrivateApis );
 
 const { Slot: InspectorSlot, Fill: InspectorFill } = createSlotFill(
 	'EditSiteSidebarInspector'
 );
 export const SidebarInspectorFill = InspectorFill;
 
+const FillContents = ( {
+	sidebarName,
+	isEditingPage,
+	supportsGlobalStyles,
+} ) => {
+	// Because `DefaultSidebar` renders a `ComplementaryArea`, we
+	// need to forward the `Tabs` context so it can be passed through the
+	// underlying slot/fill.
+	const tabsContextValue = useContext( Tabs.Context );
+
+	return (
+		<>
+			<DefaultSidebar
+				identifier={ sidebarName }
+				title={ __( 'Settings' ) }
+				icon={ isRTL() ? drawerLeft : drawerRight }
```

Comment:
Total aside, but I wish we didn't have to do this, and could instead say `icon={ drawer }` letting the icon worry about the context.